frontend/src/lib/utils.ts

```diff
@@ -46,11 +45,73 @@ export function debounce(delay: number, fn: any): any {
 }
 
 /**
- * Returns true if the URL parameters indicated that we're embedded in an
- * iframe.
+ * Param values, which can be set in ?embed={value}, all should be lowercase
  */
-export function isEmbeddedInIFrame(): boolean {
-  return url.parse(window.location.href, true).query.embed === "true"
+export const embedParamValues = new Set<string>([
+  "noColoredBar".toLowerCase(),
+  "noToolBar".toLowerCase(),
+  "noPadding".toLowerCase(),
+  "noScroll".toLowerCase(),
+  "noFooter".toLowerCase(),
+  "true".toLowerCase(),
+])
+
+/**
+ * Returns list of defined embed url params
+ */
+export function getEmbedUrlParams(): Set<string> {
+  const embedUrlParams = new Set<string>()
+  const urlParams = new URLSearchParams(window.location.search)
+  urlParams.forEach((paramValue, paramKey) => {
+    paramKey = paramKey.toString().toLowerCase()
+    paramValue = paramValue.toString().toLowerCase()
+    if (paramKey === "embed" && embedParamValues.has(paramValue)) {
```

Comment:
nit: I'd extract `embed` as a constant to avoid using magic stuff inside the function.